lib/client.js

```diff
@@ -1,29 +1,22 @@
 (function (global, factory, undefined) {
 
-  if ( typeof module === 'object' && typeof module.exports === 'object' ) {
-    var ActivityStreams = require('activity-streams');
+  if ( typeof module === "object" && typeof module.exports === "object" ) {
```

Comment:
Crockford made a case against using single quotes as the default quotation mark. Can't find a link at the moment, but more an more I'm running into eslint or tslint which complains about single quotes, so I'm trying to force myself into the habit.